### PR TITLE
 feat(TDI-48193): enable LOCAL_FILE by default in PostgreSQL bulk components

### PR DESCRIPTION
**Pull request by mhryb**
**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.repository/plugin.xml [hunk 1]
```diff
@@ -3872,6 +3872,15 @@
               name="ReplaceSQSCheckBoxWithDropDownListMigrationTask"
               version="8.0.1">
         </projecttask>
+       <projecttask
+               beforeLogon="false"
+               breaks="8.0.1"
+               class="org.talend.repository.model.migration.ChangeDefaultValueLocalFileCopyInPostgresql"
+               description="Change local file for copy checkbox in Postgresql bulk components"
+               id="org.talend.repository.model.migration.ChangeDefaultValueLocalFileCopyInPostgresql"
+               name="ChangeDefaultValueLocalFileCopyInPostgres"
```
Comment (dmytro-sylaiev) [inline main/plugins/org.talend.repository/plugin.xml:3881]: could you please name the migration job name corresponding to the class name as well?

**Review by dmytro-sylaiev — COMMENTED** [review]
LGTM, Just a small coment

File: main/plugins/org.talend.repository/plugin.xml (hunk 1 above)
Comment (mhryb) [reply main/plugins/org.talend.repository/plugin.xml:3881]: Sure, thank you for the notice!

**Review by dmytro-sylaiev — DISMISSED** [review]
LGTM

**Review by ozhelezniak-talend — APPROVED** [review]
LGTM

**Pull request merged**